Alpine doesn't have bash by default and we want to avoid dependencies on bash, if possible.
In the cases below, the scripts can be made trivially compatible with POSIX shell.

--- a/zotero/app/linux/zotero
+++ b/zotero/app/linux/zotero
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 # Increase open files limit
 #
--- a/zotero/app/linux/zotero.desktop
+++ b/zotero/app/linux/zotero.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=Zotero
-Exec=bash -c "$(dirname $(realpath $(echo %k | sed -e 's/^file:\\/\\///')))/zotero -url %U"
+Exec=sh -c "$(dirname $(realpath $(echo %k | sed -e 's/^file:\\/\\///')))/zotero -url %U"
 Icon=zotero.ico
 Type=Application
 Terminal=false
--- a/zotero/app/linux/set_launcher_icon
+++ b/zotero/app/linux/set_launcher_icon
@@ -1,10 +1,10 @@
-#!/bin/bash -e
+#!/bin/sh -e
 
 #
 # Run this to update the launcher file with the current path to the application icon
 #
 
-APPDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+APPDIR="$( cd "${0%/*}" && pwd )"
 if [ -w "$APPDIR"/zotero.desktop ]; then
 	sed -i -e "s@^Icon=.*@Icon=$APPDIR/icons/icon128.png@" "$APPDIR"/zotero.desktop
 else
